conf/parser.go

```diff
@@ -255,6 +262,12 @@ func FromWgQuick(s, name string) (*Config, error) {
 					return nil, err
 				}
 				conf.Interface.TableOff = tableOff
+			case "lan":
```

Comment:
nit: I'd follow the convention of `persistkeepalive` or `table` and call it `excludelocalnetworks` and possible values being `off` or `on`.
`lan=exclude` is not as clear I think.

Reply:
Makes sense. I've called it `excludelan` to have a shorter name and possible values are `on`/`off`